ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+puts "How many employees are being interviewed"
+interviewed = gets.to_i
+
+while interviewed >= 1
+
+puts "What is your name?"
+name = gets.chomp
+
+puts "How old are you?"
+age = gets.to_i
+
+puts "What year were you born?(yyyy)"
+year = gets.to_i
+
+puts "Would you like garlic bread from the cafeteria?(yes, no)"
+garlic_preference = gets.chomp
+
+puts "Would you like to enroll in the health insurance?(yes, no)"
+insurance = gets.chomp
+
+  if 2017 - year == age
```

Comment:
Check out `Time.new.year`, this will give the current year instead of hard-coding in a value.